database/migrations/2019_12_22_015115_create_short_urls_table.php

```diff
@@ -13,7 +13,7 @@ class CreateShortUrlsTable extends Migration
      */
     public function up()
```

Comment:
I'm not familiar with using different connections myself. Do you think we'll also need to set the `connection` field on each of the migrations to determine which of the connections to run the migrations on? 🙂